frontend/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -31,6 +31,11 @@ const ProgressBar: React.FC<ProgressProps> = ({
           pathColor: '#9be2a2',
           trailColor: '#eaeaea',
         }),
+        ...(variant === VARIANTS.notStarted && {
+          textColor: '#000000',
+          pathColor: '#eaeaea',
+          trailColor: '#eaeaea',
+        }),
         ...(variant === VARIANTS.failed && {
           textColor: '#000000',
           pathColor: '#ff9494',
```

Comment:
Please, change trailColor for Failed status (color '#eaeaea') as it shown in Figma.

Reply:
![image](https://user-images.githubusercontent.com/52101645/161637877-6a555c48-7c6f-42a7-962b-d27d885631b0.png)